activerecord-postgres_pub_sub.gemspec

```diff
@@ -41,7 +41,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.add_runtime_dependency "activerecord", ">= 5.1", "< 5.3"
-  spec.add_runtime_dependency "pg", "~> 0.18"
+  spec.add_runtime_dependency "pg", ">= 0.18", "< 1.2"
```

Comment:
This should allow dependent apps to use anything between 0.18.x to 1.1.x, so I think apps currently using this gem shouldn't be upgrading their pg gems unless the pg gem is being explicitly upgraded in the app.